alexis.py

```diff
@@ -356,6 +365,20 @@ def init(self):
             resp = 'Hay {} {}:\n```\n{}\n```'.format(num_memes, word, '\n'.join(memelist))
             await self.send_message(chan, resp)
 
+        #cleverbot (@bot <mensaje>)
+        elif text.startswith('<@{}>'.format(self.user.id)):
```

Comment:
El bot se inicia, pero... que ocurre cuando se llama al "comando"? ¿El bot respondería "No ando con ganas de hablar."? ¿No será mejor simplemente desactivar el "comando"? -> Esto implica agregar `and self.cbotcheck is not None` en la condición de esta línea.